crowbar_framework/app/controllers/api/restart_management_controller.rb

```diff
@@ -130,6 +138,7 @@ def configuration_post
 
   def get_node_or_raise(node_name)
     node = NodeObject.find("name:#{node_name}").first
+    node = NodeObject.find_node_by_alias(node_name) if node.nil?
```

Comment:
You should use `NodeObject.find_node_by_name_or_alias`

Reply:
It was my first attempt, but It fails in some travis tests but not in my command line tests. So I change for find_node_by_alias and it works in travis tests, and in my command line tests. If I change it I should or modify find_node_by_name_or_alias to accept or this code to accept exceptions. However I've already tested it without good results. Let me know if knowing this information I have to change it anyway, plese

Reply:
I'd still do the change and try to figure out why travis fails. Else, it's a pattern that people will repeat and it's not what we want :-)

Reply:
Ok, thankyou, WIP

Reply:
@ilausuch the reason that this may fail to pass the tests may be because the `NodeObject.find_node_by_alias(node_name)` may not be mocked properly, so its expected to fail or not return the expected status.

Hit me up for a joint session on the testing stuff if you need some help to dig on that further :)

P.S.: Also there should be no need for 2 different node searches there, Im guessing you wanted to remove the first one.